samples/client.py

```diff
@@ -90,7 +90,7 @@ async def process_stream(self, prompts, sampling_parameters):
                     self._results_dict[result.get_response().id].append(i)
 
     async def run(self):
-        sampling_parameters = {"temperature": "0.1", "top_p": "0.95"}
+        sampling_parameters = {"temperature": "0.1", "top_p": "0.95", "max_tokens":"100"}
```

Comment:
Thanks for this PR! Our `client.py` serves as a sample client, thus maybe it makes sense to put some comments above and not an actual code

Reply:
@oandreeva-nv We can add documentation to the README for the currently supported sampling parameters and/or provide command line arguments to set them in the sample client. Realized we don't currently document the list of supported sampling parameters -

Reply:
`supported sampling parameters` are actively changing. I would avoid specifying list in README ATM